src/core-components/SwapInput/SwapInput.tsx

```diff
@@ -0,0 +1,57 @@
+import './SwapInput.less';
+
+import React from 'react';
+
+import { escapeRegExp } from '../../utils/format';
+import { Input } from '../index';
+
+const inputRegex = RegExp(`^\\d*(?:\\\\[.])?\\d*$`); // match escaped "." characters via in a non-capturing group
+
+interface SwapInputProps {
+  value: string | number;
```

Comment:
We will use only `strings` to display `values`.
We have to use only one type of inputs to avoid future confusion. If we will use number to display values it may cause unexpected errors related to the javascript language. like `0.04 == 0.04000000002`
SDK waits for `bigint` types, so after request, we will have to transform all values to `bigint`. And if the value will be wrong we will debit more money from the user's account.

Reply:
where is the InputToken component?